### PR TITLE
dev: Dark mode support for alerts

### PR DESCRIPTION
**Pull request by abbyoung**
# SC-1177

## Proposed changes

This PR adds portal dark theme support for the React USWDS Alert component. 

* Theme variables to represent current react uswds defaults (light mode)
* Theme variables to represent ussf dark mode colors
* CSS to implement in `index.scss`
* Storybook Alert stories for each type of alert (success, warning, info, error)

## Reviewer notes

There are a few places we use the alert component: 

* When editing display name, you should see a Success Alert on save.
* Once you hit 9 out of 10 links in a collection, you should see a Warning Alert when creating a custom link.
* Add a bookmark from Sites & Apps alpha view, and you should see a Success alert.

To view other alert states (error, info), you can run storybook `yarn storybook` and check out the alerts in light and dark mode.

## Setup

You can either run the app and cms locally, or view the alert component on Storybook and use the theme toggle.

---

### As the original developer, I have

- [ ] Met the acceptance criteria, or will meet them in subsequent PRs or stories
  - ... 
- [ ] Created new stories in Storybook if applicable
  - [ ] Checked that all Storybook accessibility checks are passing
- [ ] Created/modified automated unit tests in Jest
  - [ ] Including jest-axe checks when UI changes
- [ ] Created/modified automated E2E tests in Cypress
  - [ ] Including cypress-axe checks when UI changes
  - [ ] Checked that the E2E test build is not failing
- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):
  - [ ] Checked responsiveness in mobile, tablet, and desktop
  - [ ] Checked keyboard navigability
  - [ ] Tested with [VoiceOver](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) in Safari
  - [ ] Checked VO's [rotor menu](https://github.com/trussworks/accessibility/blob/master/README.md#how-to-use-the-rotor-menu) for landmarks, page heading structure and links
  - [ ] Used a browser a11y tool to check for issues (WAVE, axe, ANDI or Accessibility addon tab for Storybook)
- [ ] Requested a design review for user-facing changes
- For any new migrations/schema changes:
  - [ ] Followed guidelines for zero-downtime deploys

### As code reviewer(s), I have

- [ ] Pulled this branch locally and tested it
- [ ] Reviewed this code and left comments
- [ ] Checked that all code is adequately covered by tests
  - [ ] Checked that the E2E test build is not failing
- [ ] Made it clear which comments need to be addressed before this work is merged
- [ ] Considered marking this as accepted even if there are small changes needed
- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):
  - [ ] Checked responsiveness in mobile, tablet, and desktop
  - [ ] Checked keyboard navigability
  - [ ] Tested with [VoiceOver](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) in Safari
  - [ ] Checked VO's [rotor menu](https://github.com/trussworks/accessibility/blob/master/README.md#how-to-use-the-rotor-menu) for landmarks, page heading structure and links
  - [ ] Used a browser a11y tool to check for issues (WAVE, axe, ANDI or Accessibility addon tab for Storybook)

### As a designer reviewer, I have

- [ ] Checked in the design translated visually
- [ ] Checked behavior
- [ ] Checked different states (empty, one, some, error)
- [ ] Checked for landmarks, page heading structure, and links
- [ ] Tried to break the intended flow
- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):
  - [ ] Checked responsiveness in mobile, tablet, and desktop
  - [ ] Checked keyboard navigability
  - [ ] Tested with [VoiceOver](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) in Safari
  - [ ] Checked VO's [rotor menu](https://github.com/trussworks/accessibility/blob/master/README.md#how-to-use-the-rotor-menu) for landmarks, page heading structure and links
  - [ ] Used a browser a11y tool to check for issues (WAVE, axe, ANDI or Accessibility addon tab for Storybook)

### As a test user, I have

- Run through the [Test Script](hhttps://docs.google.com/spreadsheets/d/1eV8UK0aJZ0qrzjnXf5SJ_uRiV7bpsj3yrhRFEkjOvV4/edit?usp=sharing):
  - [ ] On commercial internet in IE11
  - [x] On commercial internet in Edge
  - [x] On commercial internet in Firefox
  - [x] On commercial internet in Chrome
  - [x] On commercial internet in Safari
  - [ ] On NIPR in IE11
  - [ ] On NIPR in Firefox
  - [ ] On NIPR in Chrome
  - [ ] On NIPR in Safari
  - [ ] On a mobile device in Firefox
  - [ ] On a mobile device in Chrome
  - [ ] On a mobile device in Safari
- [ ] Added any anomalous behavior to this PR

---

### CODE REVIEW

**Review by jbecker01 — APPROVED** [review]
Looks good!

**Review by jcbcapps — APPROVED** [review]

File: .storybook/preview.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { MockedProvider } from '@apollo/client/testing'
 // happo support
 import 'happo-plugin-storybook/register'
 
+import 'uswds/dist/css/uswds.css'
```
Comment (abbyoung) [inline .storybook/preview.js:8]: By adding this import, all of the missing icons now appear 🌟 That includes all of the little black boxes in the News widget RSS feed

File: .storybook/main.js [hunk 2]
```diff
@@ -25,17 +25,21 @@ module.exports = {
...
-        { loader: 'css-loader', options: { url: false } },
+          // All CSS assets will be served from /public so no need to resolve URLs
+          { loader: 'css-loader', options: { url: false } },
 
-        { loader: 'sass-loader', options: { warnRuleAsWarning: false } },
-      ],
-    })
+          { loader: 'sass-loader', options: { warnRuleAsWarning: false } },
+        ],
+      })
```
Comment (gidjin) [inline .storybook/main.js:42]: Suggestion: looks like the semi-colon and parens around the new alias config are not required and indented the rules unnecessarily. I don't need to re-review after you update that.

**Review by gidjin — APPROVED** [review]
See my one suggestion in `.storybook/main.js` but looks good otherwise.

File: .storybook/main.js (hunk 2 above)
Comment (abbyoung) [reply .storybook/main.js:42]: my code editor keeps adding them based on linter rules...why 😩

**Pull request merged**